index.js

```diff
@@ -1,18 +1,34 @@
 var LaunchDarkly = require('@launchdarkly/node-server-sdk');
 
 // Set sdkKey to your LaunchDarkly SDK key.
-const sdkKey = "";
+const sdkKey = process.env.LAUNCHDARKLY_SERVER_KEY;
 
 // Set featureFlagKey to the feature flag key you want to evaluate.
-const featureFlagKey = "my-boolean-flag";
+const featureFlagKey = typeof process.env.LAUNCHDARKLY_FLAG_KEY !== "undefined" ? process.env.LAUNCHDARKLY_FLAG_KEY : "sample-feature";
+
+const CI = typeof process.env.CI !== "undefined";
+
+function showBanner() {
+  console.log(
+`        ██
+          ██
+      ████████
+         ███████
+██ LAUNCHDARKLY █
+         ███████
+      ████████
+          ██
+        ██
+`)
+}
 
-function showMessage(s) {
-  console.log("*** " + s);
-  console.log("");
+function printValueAndBanner(_, flagValue) {
+    console.log("*** Feature flag '" + featureFlagKey + "' is " + flagValue + " for this context\n");
```

Comment:
```suggestion
    console.log("*** The '" + featureFlagKey + "' feature flag evaluates to " + flagValue + ".\n");
```
to match https://github.com/launchdarkly/sdk-specs/tree/main/specs/EXAM-SDK-example/HELLO-hello-apps#requirement-142